package.json

```diff
@@ -15,7 +15,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/heroku/react-refetch.git"
+    "url": "git+https://github.com/heroku/react-refetch.git"
```

Comment:
What is this change for?

Reply:
This was automatic as part of the pre-prep. See https://docs.npmjs.com/cli/v10/configuring-npm/package-json#git-urls-as-dependencies. Unclear as this shouldn't be valid for the `repository` object. I'll remove.